pkg/commands/workload_apply_test.go

```diff
@@ -265,7 +274,9 @@ To get status: "tanzu apps workload get my-workload"
 					ObjectMeta: metav1.ObjectMeta{
 						Namespace: defaultNamespace,
 						Name:      workloadName,
-						Labels:    map[string]string{},
+						Labels: map[string]string{
+							apis.WorkloadTypeLabelName: "web",
```

Comment:
Is it possible to validate that , update to an existing workload doesn't add the type or modify the type via apply or update?

Reply:
added test case, pointed in a different comment